src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
Browsers cannot interpret `.scss` files directly. Please compile your Sass file to a CSS file and update this link to point to the compiled CSS file instead.